chapter_deep-learning-computation/model-construction.md

```diff
@@ -534,6 +634,22 @@ chimera.add(FixedHiddenMLP())
 chimera(X)
 ```
 
+```{.python .input  n=8}
+#@tab paddle
+class NestMLP(nn.Layer):
+    def __init__(self):
+        super().__init__()
+        self.net = nn.Sequential(nn.Linear(20, 64), nn.ReLU(),
+                                 nn.Linear(64, 32), nn.ReLU())
+        self.linear = nn.Linear(32, 16)
```

Comment:
I am wondering if paddle provides interface like lazylinear in pytorch.

Reply:
PaddlePaddle don't provide lazy API like pytorch, I have submitted a issue for this API, https://github.com/PaddlePaddle/Paddle/issues/42703